.github/workflows/eden_rpi.yml

```diff
@@ -0,0 +1,140 @@
+---
+name: EdenRPi4
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows:
+      - Publish
+    types:
+      - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        hv: ["kvm", "xen"]
+    steps:
+      - name: Check secrets
+        run: |
```

Comment:
Shouldn't this run block be after env block below @Insei ?

Reply:
No, this is not important, for expamples:
success: https://github.com/Insei/eve/runs/4683429604
failed: https://github.com/Insei/eve/actions/runs/1638284468
All works fine